resources/user-agents/browsers/firebird/firebird-0-x.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Firebird is what Firefox was called before they changed the name to Firefox. I extracted the rules from the firefox-0-x.json file that were for Firebird and put them here, mainly because Firebird's versions were only 0.6 and 0.7 AFAICT. 0.8 was the first release that had the Firefox name. This is discussed more here: https://en.wikipedia.org/wiki/History_of_Firefox#Naming